File: Swift/프로그래머스/0/181942. 문자열 섞기/문자열 섞기.swift
func solution(_ str1: String, _ str2: String) -> String {
    var result = ""
    
    let length1 = str1.count
    let length2 = str2.count
    let maxLength = max(length1, length2)
    
    for i in 0..<maxLength {
        if i < length1 {
            let index = str1.index(str1.startIndex, offsetBy: i)
            result.append(str1[index])
        }
        if i < length2 {
            let index = str2.index(str2.startIndex, offsetBy: i)
            result.append(str2[index])
        }
    }
    
    return result
}

File: Java/프로그래머스/0/181942. 문자열 섞기/문자열 섞기.java
public class Solution {
    public String solution (String str1, String str2) {
        
        StringBuilder sb = new StringBuilder();
        
        int len = str1.length();
        
        for (int i = 0; i < len; i++) {
            sb.append(str1.charAt(i));
            sb.append(str2.charAt(i));
        }
        
        return sb.toString();
    }
}